tests/fixtures/browser.ts

```diff
@@ -43,36 +43,27 @@ import { LoadingState } from 'src/shared/types/loading-state';
 import { BreakpointWidth } from 'src/global/globalConfig';
 
 export const createCogTrackList = (): CogList => ({
-  'track:contig': faker.datatype.number(),
-  'track:gc': faker.datatype.number(),
-  'track:gene-feat': faker.datatype.number(),
-  'track:gene-other-fwd': faker.datatype.number(),
-  'track:gene-other-rev': faker.datatype.number(),
-  'track:gene-pc-fwd': faker.datatype.number(),
-  'track:gene-pc-rev': faker.datatype.number(),
-  'track:variant': faker.datatype.number()
+  'gene-focus': faker.datatype.number(),
+  contig: faker.datatype.number(),
+  gc: faker.datatype.number()
 });
 
-export const createTrackConfigLabel = () => ({
-  'track:contig': true,
-  'track:gc': true,
-  'track:gene-feat': true,
-  'track:gene-other-fwd': true,
-  'track:gene-other-rev': true,
-  'track:gene-pc-fwd': true,
-  'track:gene-pc-rev': true,
-  'track:variant': true
-});
-
-export const createTrackConfigNames = () => ({
-  'track:contig': true,
-  'track:gc': true,
-  'track:gene-feat': true,
-  'track:gene-other-fwd': true,
-  'track:gene-other-rev': true,
-  'track:gene-pc-fwd': true,
-  'track:gene-pc-rev': true,
-  'track:variant': true
+export const createTracksInfo = () => ({
+  'gene-focus': {
+    showMoreTranscripts: false,
+    showTranscriptIds: false,
+    showTrackName: true,
+    showFeatureLabel: true,
+    trackType: 'gene'
+  },
```

Comment:
I mentioned creation of functions that return default configs for different track types in a comment to trackConfigSlice. You may want to reuse them here.

Reply:
✅